worlds/shivers/__init__.py

```diff
@@ -166,11 +213,13 @@ def pre_fill(self) -> None:
     def fill_slot_data(self) -> dict:
 
         return {
-            "storageplacements": self.storage_placements,
-            "excludedlocations": {str(excluded_location).replace('ExcludeLocations(', '').replace(')', '') for excluded_location in self.multiworld.exclude_locations.values()},
-            "elevatorsstaysolved": {self.options.elevators_stay_solved.value},
-            "earlybeth": {self.options.early_beth.value},
-            "earlylightning": {self.options.early_lightning.value},
+            "StoragePlacements": self.storage_placements,
+            "ExcludedLocations": {str(excluded_location).replace('ExcludeLocations(', '').replace(')', '') for excluded_location in self.multiworld.exclude_locations.values()},
```

Comment:
If my understanding is correct, we only actually care what locations in this slot are excluded so we can just do this.

```suggestion
            "ExcludedLocations": list(self.options.exclude_locations.value),
```

Reply:
Changed